headers/addons/analog.h

```diff
@@ -69,6 +69,10 @@
 #define SMOOTHING_FACTOR 5
 #endif
 
+#ifndef ANALOG_ERROR
+#define ANALOG_ERROR 0
```

Comment:
If the default value is zero we need to modify form [validation](https://github.com/OpenStickCommunity/GP2040-CE/pull/1136/files#diff-1dc51191c1aed5b94ed7849f6d8458038a17ed0c47da979dd4de7e7adc3a8c99R106)
As zero does not exist as any value here: https://github.com/OpenStickCommunity/GP2040-CE/pull/1136/files#diff-1dc51191c1aed5b94ed7849f6d8458038a17ed0c47da979dd4de7e7adc3a8c99R25

Otherwise we will get the error:
<img width="353" alt="Screenshot 2024-09-16 at 22 09 03" src="https://github.com/user-attachments/assets/5e90bafa-ed24-4f74-907e-ea1e43be6055">

When enabling switch the first time.